components/calendar/src/error.rs

```diff
@@ -60,14 +60,18 @@ pub enum CalendarError {
 
 /// A list of error outcomes for exceeding location bounds
 #[derive(Display, Debug, Copy, Clone, PartialEq)]
-pub enum LocationError {
+pub enum LocationOutOfBoundsError {
```

Comment:
issue: this enum should be pub(crate) I think

Reply:
Done